assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -161,6 +161,11 @@ che.docker.always_pull_image=true
 # to be able to launch their own runtimes which are embedded Docker containers.
 che.docker.privileged=false
 
+# This parameter allows to specify custom security options for the created docker container
+# seccomp:unconfined is the default for kubernetes, but not for docker. This is needed
+# for debugging with gdbserver
+che.docker.securityopt=seccomp:unconfined
```

Comment:
Do we want to change default settings or just allow to change it? @hkolvenbach @tolusha

Reply:
I would say yes to behave in the same way as the kubernetes / openshift implementation does. Any comments on this @tolusha?